### ISSUE TITLE
Backlink hover should show more useful information

### ISSUE THREAD

**Issue by Tyriar** [lostintangent/wikilens#9]
![image](https://user-images.githubusercontent.com/2193314/130459761-55333921-5645-4b52-b179-b223baef7fae.png)

Perhaps showing the path would more useful? Something like

> Home > Home Improvements
>
> `Home/home-improvements.mds`

or

> Home Improvements
>
> Under `Home`

**Comment by Tyriar** [comment]
Just realized it's peeking the contents in https://github.com/lostintangent/wikilens/issues/8

Still breadcrumbs might be useful above the peek?

```
`Home > Home Improvements`

## Home Improvements

<content>
```

**Comment by lostintangent** [comment]
Yep agreed! I'll add the path breadcrumb to the top, since I really like that suggestion.

**Comment by lostintangent** [comment]
I just shipped `0.0.4`, which changes the tooltip to display the page name and line preview, as key/value pairs. That way, the tooltip more accurately reflects the title/description of the node. Let me know if you have any further feedback. Thanks again!
